src/QAT/qat/purr/compiler/instructions.py

```diff
@@ -396,7 +392,10 @@ def __init__(self, acquire: Acquire, process, axes=None, args=None):
             axes = [axes]
 
         self.process: PostProcessType = process
-        self.args: List = args or []
+
+        # TODO: Should find out why ags can't reference model objects in builder serialization.
```

Comment:
can we put an issue in for this as a bug, cheers

Reply:
Done. https://github.com/oqc-community/qat/issues/86